File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
       vector<int>count;
       int temp=0;
       for(int i=0;i<nums.size();i++){
        
        if(nums[i]==1){
            temp++;
        }
        else if(nums[i]==0){
            count.push_back(temp);
            temp=0;
        }
        count.push_back(temp);
       } 
       int result=*max_element(count.begin(),count.end());
       return result;
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int l=0,r=1;
        int count=0;
        for(r=1;r<nums.length;r++){
            if(nums[l]!=nums[r]){
                l++;
                nums[l]=nums[r];
                
            }            
        }
        return l+1;
    }
}